src/JsonApiDotNetCore.MongoDb/JsonApiDotNetCore.MongoDb.csproj

```diff
@@ -1,12 +1,29 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <VersionPrefix>4.0.0-rc</VersionPrefix>
```

Comment:
Yes, it was me who published the package. I did it because the PR had been stale for a while but I'll be more than happy to transfer ownership of it to you.

Reply:
Thanks, I would like to accept ownership of it. I can look into setting up cibuild somewhere next week. We've been having lots of timeouts with Travis CI recently, so I would prefer to use AppVeyor which is always fast. Also that enables me to reuse most of the existing scripts.